projects/helm/chartmuseum/Makefile

```diff
@@ -0,0 +1,35 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=chartmuseum
+REPO_OWNER=helm
+
+BASE_IMAGE_NAME?=eks-distro-minimal-base
+SOURCE_PATTERNS=./cmd/chartmuseum
+
+BINARY_TARGET_FILES=chartm
+
+HAS_S3_ARTIFACTS=true
+IMAGE_NAMES=
+
+PRECOMPILE_PREPARE=harbor_patch
+
+HARBOR_VERSION=v2.4.1
+
+harbor_patch:
+	cd $(PROJECT_ROOT) \
+	&& git clone https://github.com/goharbor/harbor.git harbor \
+	&& git -C harbor checkout -f $(HARBOR_VERSION) \
+	&& cd $(REPO) && git apply ../harbor/make/photon/chartserver/redis.patch \
```

Comment:
I think i would instead pull this into our repo in a patches directory here and let it apply like we do other patches to avoid this part.  A bit of duplication, but I think that would be cleaner and avoid additional clones that we may have to worry about when we are thinking about pulling everything from codecommit in codebuild.